client/components/domains/connect-domain-step/connect-domain-step-ownership-auth-code.jsx

```diff
@@ -28,6 +29,14 @@ const ConnectDomainStepOwnershipAuthCode = ( {
 				) }
 			</p>
 			<p className="connect-domain-step__text">{ authCodeStepDefaultDescription.label }</p>
+			<p className="connect-domain-step__text">
+				{ createInterpolateElement(
+					__(
+						'This will only be used for verification purposes, a domain transfer <strong>will not</strong> be initiated.'
```

Comment:
What do you think about this slightly different message: "This will only be used to verify that you own this domain; we won't transfer it." Or something similar. (I was just trying be more specific about the reason for the verification and get rid of the word "initiated".)

Reply:
I like that, it sounds more direct 👍